### PR TITLE
Win32 install

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
preparing a win32 support

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
this replaces #30

File: pipsi.py [hunk 1]
```diff
@@ -89,7 +122,7 @@ def find_installed_executables(self, path):
 
     def find_scripts(self, virtualenv, package):
         prefix = os.path.normpath(os.path.realpath(os.path.join(
-            virtualenv, 'bin'))) + '/'
+            virtualenv, BIN_DIR))) + '/'
```
Comment (untitaker) [inline pipsi.py]: Replace ``"/"`` with os.path.sep.

**Comment by untitaker** [comment]
Line 31: os.readlink doesn't exist.

**Comment by RonnyPfannschmidt** [comment]
https://devpi.net/ronny/dev/pipsi/0.9.dev0 the testresults ^^

**Comment by fpatz** [comment]
I had to add `os.path.normpath()` to the path normalization to make it work for me, otherwise prefix matching did not work and it didn't find the scripts to link/copy. Do you want a PR?

**Comment by RonnyPfannschmidt** [comment]
yes please, i tested on my systems and it worked, but i had the suspicion there might have been a bug, did the tests fail for you, or is a new one needed?

**Comment by RonnyPfannschmidt** [comment]
maybe we do need a test package that exposes the different kinds of scripts (py, pyw, entrypoints)

**Comment by RonnyPfannschmidt** [comment]
@mitsuhiko ping?

**Pull request merged**